src-docs/src/views/breadcrumbs/color.tsx

```diff
@@ -7,9 +7,7 @@ const breadcrumbs: EuiBreadcrumb[] = [
     text: 'Animals',
     href: '#',
     color: 'primary',
-    onClick: (e: { preventDefault: () => void }) => {
-      e.preventDefault();
-    },
+    onClick: (e) => e.preventDefault(),
```

Comment:
TS has enough info to not need an explicit type in these cases

Reply:
Hm, my IDE complained about `e`... maybe because of the braces?